### ISSUE TITLE
Realtime changes in wave conditions in response to converter 

### ISSUE THREAD

**Issue by Wavergy** [WEC-Sim/WEC-Sim#209]
Hi Wec-Sim

I had a general question about the capability of wec-sim to both visually show and compute effects of a wave energy converter on wave conditions in real time, and use those changes in wave conditions as a factor in its analysis.

For example, let's say there is a very long wave energy converter that is highly efficient at its front where it first touches a wave crest - therefore reducing the wave height significantly as it "drains" energy from a wave, with another pto towards the device's rear that captures some of that remaining wave energy.  This is hypothetical, of course.

Will we visually see these theoretical two drops in wave height, and will that rear pto in the device show accurate results representing the fact that it is interacting with a lower height (energy dense) wave after the wave has passed through part of the device already?

In summary : Is there that realtime feedback loop in wec -sim  whereby wave conditions change as they pass through a device - with those new conditions available to be captured then in real time ?

**Comment by kmruehl** [comment]
@Wavergy, WEC-Sim is developed based on linear wave theory, assuming linear superposition of the wave field. In WEC-Sim, we do not model the fluid domain directly. Instead, we model the wave loading on the body based on the Cummins' equation. As such, WEC-Sim does not model the change in the wave field due to the WEC's energy extraction. 

For more info, please refer to the [theory section](http://wec-sim.github.io/WEC-Sim/theory.html#) of our documentation.
